core/app/modules/import-export/assets/js/pages/import/import-complete/components/import-complete-footer/import-complete-footer.js

```diff
@@ -12,15 +12,41 @@ export default function ImportCompleteFooter( { seeItLiveUrl } ) {
 				<Button
 					text={ __( 'See it live', 'elementor' ) }
 					variant="contained"
-					onClick={ () => window.open( seeItLiveUrl, '_blank' ) }
+					onClick={ () => {
+						if ( 'kit-library' === referrer ) {
```

Comment:
the IE shouldn't know about the kit library

Reply:
As I see it I must add a Literal as there is no real connection between these apps.
Have we encountered a similar use case?